controllers/chaincodebuild/predict.go

```diff
@@ -123,9 +125,18 @@ func (r *ReconcileChaincodeBuild) patchChaincodeBuildStatus(build *current.Chain
 		if condition.Reason == pipelinev1beta1.PipelineRunReasonCompleted.String() && condition.Status == corev1.ConditionTrue {
 			// retrieve pipeline results
 			if newStatus.PipelineResults != nil {
+				if build.Status.PipelineRunResults == nil {
+					build.Status.PipelineRunResults = make([]pipelinev1beta1.PipelineRunResult, 0)
```

Comment:
I think the judgment here is not needed.
The following assignment statement ensures that the field is not nil.